examples/example-web/src/composeServer.ts

```diff
@@ -16,6 +18,8 @@ export default async (): Promise<WebServerComposition> => {
   ROOT_CONTAINER.register('appName', {useValue: appName});
   ROOT_CONTAINER.register('serverLogFile', {useValue: 'server.log'});
   ROOT_CONTAINER.register('logLevel', {useValue: 'debug'});
+  ROOT_CONTAINER.register('customAuthWhitelist', {useValue: []});
```

Comment:
Let's remove this line and make sure things work without it.

Reply:
Removed